src/redux/actionCreators.js

```diff
@@ -18,24 +18,42 @@ const updateNewMessageBodyCreator = (payload) => {
 
 const updateSendMessageCreator = () => ({ type: ACTION_TYPES.SEND_MESSAGE });
 
-const updateFollowStatusAC = (payload) => ({
+const follow = (payload) => ({
   type: ACTION_TYPES.FOLLOW_TO_USER,
   payload,
 });
 
-const updateUnfollowStatusAC = (payload) => ({
+const unfollow = (payload) => ({
   type: ACTION_TYPES.UNFOLLOW_TO_USER,
   payload,
 });
 
-const setUsersAC = (users) => ({ type: ACTION_TYPES.SET_USERS, users });
+const setUsers = (users) => ({ type: ACTION_TYPES.SET_USERS, users });
+
+const setCurrentPage = (currentPage) => ({
+  type: ACTION_TYPES.SET_CURRENT_PAGE,
+  currentPage,
+});
+
+const setTotalUsersCount = (totalUsersCount) => ({
+  type: ACTION_TYPES.SET_TOTAL_USERS_COUNT,
+  totalUsersCount,
+});
+
+const toggleIsFetching = (isFetching) => ({
+  type: ACTION_TYPES.TOGGLE_IS_FETCHINE,
+  isFetching,
```

Comment:
Consider using "payload" as a key in actions.

Reply:
done